run-tests

```diff
@@ -0,0 +1,2 @@
+docker-compose run --rm edge make test-ci
```

Comment:
the `Makefile` is designed as the single point of entry for tasks; this script should just be a target in the makefile, `test-all`, that calls the other two targets.

Reply:
This is maybe something we need to fix for the makefile but currently using `docker-compose run --rm edge make test-ci` is the way to do test when you already have the container running. Since invoking `make test-all-ext` will start rebuilding all the images and invoking `make test-all-ext_fast` will fail if you already have a container running. I could stop the container and use `make test-all-ext_fast` which will work though.

Reply:
yeah, we should fix the Makefile. the issue is the `--service-ports` flag: we need it to run the webserver via an -ext task, but that means that when you try to run the same docker-compose service twice in different containers, the ports are already occupied.

I'll submit a fix to this branch.